src/app/plugins/dog.plugin.ts

```diff
@@ -1,66 +1,134 @@
+import { MessageEmbed } from 'discord.js';
 import Constants from '../../common/constants';
 import { Plugin } from '../../common/plugin';
-import { ChannelType, IContainer, IHttpResponse, IMessage } from '../../common/types';
+import { ChannelType, IContainer, IHttpResponse, IMessage, Maybe } from '../../common/types';
 
 export class DogPlugin extends Plugin {
   public name: string = 'Dog Plugin';
   public description: string = 'Generates pictures of doggos.';
-  public usage: string = 'dog <breed>';
+  public usage: string = 'dog <breed (Optional)> | dog breed | dog subBreed | dog random';
```

Comment:
We probaly should decide for a standard on commands cause even in my plugins I variate :scream:.  my thought here was <> implies a variable is to be used here, otherwise that is what you are meant to type literally.

That is just how my mind thought of it at this point.  If we go ahead and form a standard then I will change it here and, make a seperate pull to standardise all of them.

Reply:
The standard was that we warp everything in `<>`
Whether it be literal or a variable input. That way the number of `<>` matches the number of args